src/PhpSpec/Listener/NamedConstructorNotFoundListener.php

```diff
@@ -79,6 +79,17 @@ public function afterSuite(SuiteEvent $event)
                     'arguments' => $arguments
                 ));
                 $event->markAsWorthRerunning();
+
+                if (!method_exists($classname, '__construct')) {
```

Comment:
It might not be appropriate, but isn't that why the tool asks rather than just doing it?